server/models/article.js

```diff
@@ -5,8 +5,12 @@ export default (sequelize, DataTypes) => {
       allowNull: false,
       unique: true
     },
+    categoryId: {
+      type: DataTypes.INTEGER,
+      allowNull: false
+    },
     title: {
-      type: DataTypes.STRING,
+      type: DataTypes.INTEGER,
```

Comment:
@Veraclins I think you made a typo error here, the title in the article.js is a string, not an integer

Reply:
Thanks for pointing this out. This is obviously a mistake, I will correct it immediately.